.travis.yml

```diff
@@ -2,12 +2,23 @@ language: python
 python:
 - '3.6'
 cache:
-- pip
+  directories:
+  - $HOME/.cache/pip
+  - .arca/cache
 install:
 - pip install -r test_requirements.txt
+# https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
+# before_script:
+# - docker login -u "$DOCKER_HUB_USERNAME" -p "$DOCKER_HUB_PASSWORD"
```

Comment:
what happens on future pull-requests once this is not commented out?

Reply:
Yes, I thought about it. I'll introduce a new feature to Arca which will disable pushing to the registry but will keep pulling images. Then I'll conditionally either login to docker on master builds or turn on the "pull only" functionality of Arca in pull requests.